docs/source/algorithms/ConvertPeaksWorkspace-v1.rst

```diff
@@ -0,0 +1,56 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm provides bi-directional conversion bewteen regular PeaksWorkspace and LeanElasticPeaksWorkspace.
+
+When casting a PeaksWorkspace into a LeanElasticPeaksWorkspace, information related to detector
+(DetID, BankName, Row, Col) is discarded.
+However, other experiment information are retained in the LeanElasticPeaksWorkspace, including
+run, hkl, goniometer, sample, as well as the corresponding information at the invidual Peak level.
+
+When casting a LeanElasticPeaksWorkspace into a PeaksWorkspace, a donor PeaksWorkspace with proper
+instrument configuration is required.
+Incorrect goniometer setting will lead to a Peak with negative wavelength, which will be excluded from the output
+PeaksWorkspace.
```

Comment:
You can add links to the PeaksWorkspace and LeanElasticPeaksWorkspace concepts pages.

Reply:
I am not familiar with this part.  Could you send me an example on how to link concepts pages?  Thanks